src/Components/Movies.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+
+export class Movies extends Component {
+    constructor(){
+        super()
+
+        this.state ={
+            films: [],
+            selectedFilm: {},
+            value: ""
+        }
+    }
+
+    componentDidMount () {
+        this.getFilms()
+    }
+
+    getFilms = async () => {
+        const {data} = await axios.get(`https://ghibliapi.herokuapp.com/films`)
+        // console.log(data)
+        this.setState({
+            films: data,
+        })
+    }
+
+    handleChange = async (event) => {
+        // console.log(event.target.value)
+        this.setState({
+            value: event.target.value,
+        })
+
+        const {data} = await axios.get(`https://ghibliapi.herokuapp.com/films/${event.target.value}`)
+        // console.log(data)
+
+        this.setState({
+            selectedFilm: data,
+        })
+       
+    }
+
+    render() {
+        const {films, selectedFilm, value} = this.state;
+        // console.log(films)
+
+        const options = films.map((film,i) => 
+            <option value={film.id} key={i}>{film.title}</option>)
+        // console.log(options)
+
+        return (
+            <div>
+                <h1>Select a Movie</h1>
+                <select onChange={this.handleChange} className="Select">
```

Comment:
This `<select>` is an uncontrolled component since its `value` prop isn't controlled by react. Use `this.state.value` to control the `value` prop of the select tag!